test/test-applications/integrations/Samples.MySql/Program.cs

```diff
@@ -42,7 +53,7 @@ private static MySqlConnection OpenConnection()
                 if (oldMySqlServer)
                 {
                     var host = Environment.GetEnvironmentVariable("MYSQL57_HOST") ?? "localhost";
-                    var port = Environment.GetEnvironmentVariable("MYSQL57_PORT") ?? "3307";
+                    var port = Environment.GetEnvironmentVariable("MYSQL57_PORT") ?? "3407";
```

Comment:
Was this the wrong port?

Reply:
Yes, by default in our `docker-compose.yml`, we expose the `mysql57` service on localhost port 3407

Reply:
Was this working before then? Maybe we don't need the two MySQL instances?

Reply:
This was relatively new so I don't think the fallback was working before. I'm not sure why we have two different MySql instances, @tonyredondo would know

Reply:
There was this, but I haven't looked into it any further: https://github.com/DataDog/dd-trace-dotnet/pull/1184#discussion_r577043869

Reply:
Ok, thanks for clarifying. I'll let it go (_for now!_)

Reply:
This is because MySql 8 is a totally different database from MySql 5.7, they changed a lot (and skipped some numbers in between).

https://severalnines.com/database-blog/moving-mysql-57-mysql-80-what-you-should-know

One of the changes is the authorization and connection mechanism, so client from different versions are not compatibles by default (maybe with a lot of configuration changes can work), so it was simpler to expose two instances from the server side.

Reply:
The port change must be a left over from the local test, integration tests doesn't fail because uses the [env-vars](https://github.com/DataDog/dd-trace-dotnet/blob/zach/fix/master-adonet-fix-main-dev/docker-compose.yml#L458-L459)